master/getting-started/kubernetes/upgrade/index.md

```diff
@@ -7,25 +7,28 @@ no_canonical: true
 
 ## Prerequisites
 
-- {{site.prodname}} {{site.data.versions[page.version].first.title}} supports upgrades from 
-  {{site.prodname}} [v2.6.4](https://github.com/projectcalico/calico/releases) or 
-  later. You must [upgrade](/v2.6/getting-started/kubernetes/upgrade) to a tagged release of 
-  {{site.prodname}} [v2.6.4](https://github.com/projectcalico/calico/releases) or 
+- {{site.prodname}} {{site.data.versions[page.version].first.title}} 
+  supports upgrades from {{site.prodname}} [v2.6.4](https://github.com/projectcalico/calico/releases) or 
+  later. You must [upgrade](/v2.6/getting-started/kubernetes/upgrade) 
+  to a tagged release of {{site.prodname}} [v2.6.4](https://github.com/projectcalico/calico/releases) or 
```

Comment:
might be easier to follow if we say that "you must upgrade to v2.6.4 prior to upgrading to v3.0.0."

The "... or later ..." doesn't make sense at this point since we only have `v2.6.4` and `v3.0.0.`. @emanic , wdyt?

Reply:
I have to agree I don't like this "or later" at all...it's confusing because v3.0.0 is technically "later" as well. I'm happy to remove it.